src/components/look/tag-filter/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { Select } from 'antd';
+import { Option } from 'antd/lib/mentions';
+import styled from 'styled-components';
+
+import { P } from '@src/component/atoms';
+
+import STYLE_TAG from '@src/data/style-tags.json';
+
+function TagFilter({ idSetter }) {
```

Comment:
```suggestion
function TagFilter({ onTagSelect }) {
```

1. 이벤트 핸들러처럼 작동하는 prop 이름은 on으로 시작하도록 변경해주세요
2. prop 타이핑 해주세요!

Reply:
5161de9